x-pack/test/functional_with_es_ssl/apps/embeddable_alerts_table/embeddable_alerts_table.ts

```diff
@@ -0,0 +1,358 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import {
+  FILTERS_FORM_ITEM_SUBJ,
+  FILTERS_FORM_SUBJ,
+  RULE_TAGS_FILTER_SUBJ,
+  SOLUTION_SELECTOR_SUBJ,
+} from '@kbn/response-ops-alerts-filters-form/constants';
+import {
+  NO_AUTHORIZED_RULE_TYPE_PROMPT_SUBJ,
+  SAVE_CONFIG_BUTTON_SUBJ,
+} from '@kbn/embeddable-alerts-table-plugin/public/constants';
+import { WebElementWrapper } from '@kbn/ftr-common-functional-ui-services';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { ObjectRemover } from '../../lib/object_remover';
+
+const DASHBOARD_PANEL_TEST_SUBJ = 'dashboardPanel';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const testSubjects = getService('testSubjects');
+  const pageObjects = getPageObjects([
+    'common',
+    'home',
+    'triggersActionsUI',
+    'header',
+    'ruleDetailsUI',
+    'dashboard',
+    'timePicker',
+  ]);
+  const browser = getService('browser');
+  const security = getService('security');
+  const retry = getService('retry');
+  const find = getService('find');
+  const supertest = getService('supertest');
```

Comment:
Question: is it intentional to execute API calls with the highest possible privileges? `supertest` has system indexes access and probably not what customer will use for public APIs.

Reply:
Hey @dmlemeshko, thanks for taking a look so quickly! 🙂

Yes, that's intentional: we often prepare dependencies using elevated privileges to simplify setup in other functional tests too (see [here](https://github.com/elastic/kibana/blob/a56227bf37108da0604d731f304e0dcb81e03a4f/x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/details.ts#L30) for example). Since the rules endpoint authorization is out of scope for this test, we just need a straightforward way to generate alerts to test the embeddable panel behavior.